lib/src/http_body.dart

```diff
@@ -107,18 +108,20 @@ class HttpBodyHandler
   HttpBodyHandler({Encoding defaultEncoding = utf8})
       : _transformer = HttpBodyHandlerTransformer(defaultEncoding);
 
-  /// Process and parse an incoming [HttpRequest]. The returned [HttpRequestBody]
-  /// contains a `response` field for accessing the [HttpResponse].
+  /// Process and parse an incoming [HttpRequest].
+  ///
+  /// The returned [HttpRequestBody] contains a `response` field for accessing
+  /// the [HttpResponse].
   ///
-  /// See [HttpBodyHandler] constructor for more info on [defaultEncoding].
+  /// See [new HttpBodyHandler] for more info on [defaultEncoding].
```

Comment:
why the `new` here? Does this work with dartdoc?

Reply:
yes, this is the way to link to the unnamed constructor in dartdoc. I wish we had a nicer way, but this is the only way I know of to make the link work in both analysis server and dartdoc